web/olga/analytics/utils.py

```diff
@@ -42,3 +52,18 @@ def wrapper(request, *args, **kwargs):
         return HttpResponse(status=httplib.UNAUTHORIZED)
 
     return wrapper
+
+
+def get_coordinates_by_platform_city_name(city_name):
+    """
+    Gather coordinates by platform city name with Nominatim API.
```

Comment:
typo: s/Gather coordinates by platform city name with Nominatim API./Gather coordinates from Nominatim service by the platform city name./